### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'credentials')

### ISSUE THREAD

**Issue by ismaelmdesousa** [rajansaumya/node-red-contrib-google-sheets-advance#1]
Node-RED Version: 3.1.7
node-red-contrib-google-sheets-advance version: 0.0.2

After including the JSON with the Google Service Account credentials in the Creds field and performing the Deploy, the following error is returned:

TypeError: Cannot read properties of null (reading 'credentials')
![image](https://github.com/rajansaumya/node-red-contrib-google-sheets-advance/assets/15248222/1cae5ba4-7258-4744-8a73-0e25ad5cd8f4)

I was able to use Node after changing line 7 of the gSheetAdvance.js file:

From: `this.creds = JSON.parse(RED.nodes.getNode(config.creds).credentials.creds);`

To: `this.creds = JSON.parse(config.creds);`

**Comment by rajansaumya** [comment]
In the screenshot provided I see that the JSON credentials object for the service account is not proper. 
Please try validating json like this. 
![Untitled](https://github.com/rajansaumya/node-red-contrib-google-sheets-advance/assets/32300229/0366bfa1-6123-4bd1-bf59-deb02be7feed)
![image](https://github.com/rajansaumya/node-red-contrib-google-sheets-advance/assets/32300229/783778a0-4e30-4b4b-9293-1431a9b87e75)

**Comment by ismaelmdesousa** [comment]
From what I could see in your screenshot, the Creds field is a Configuration Node, but here, Creds is just a text field.

The edit button should appear next to the Creds field, right?

**Comment by rajansaumya** [comment]
Indeed, the 'Creds' field is a configuration node designed to receive a JSON credential object of the service account. Additionally, the edit button should be visible next to the 'Creds' field, allowing for the input of a Gauth JSON Object.

**Comment by miguelpdm** [comment]
The issue can be solved by installing https://flows.nodered.org/node/node-red-contrib-google-sheets
I guess something went wrong in the new version from [rajansaumya](https://github.com/rajansaumya) and the config node is not working

> From what I could see in your screenshot, the Creds field is a Configuration Node, but here, Creds is just a text field.
> 
> The edit button should appear next to the Creds field, right?

**Comment by rexwater** [comment]
Same, I need to install node-red-contrib-google-sheets for config node.

**Comment by rajansaumya** [comment]
Thanks for reporting this issue! It's been fixed in commit [be83cb4](https://github.com/rajansaumya/node-red-contrib-google-sheets-advance/commit/be83cb4b6a01073eddf653f1263141fe9dbe176a). The gauthAdv node now utilizes a credential field for secure credential management.

> Special thanks to @miguelpdm for finding a temporary workaround. 

Closing.
